controllers/front/successIFrame.php

```diff
@@ -115,7 +115,24 @@ public function postProcess()
             ));
         }
 
-        $orderStatusService->authorize($order);
+        try {
+            $orderStatusService->authorize($order);
+
+            Tools::redirect($this->getOrderConfirmationLink($cartId, $moduleId, $orderId, $secureKey));
+        } catch (Exception $exception) {
+            $this->warning[] = $this->module->l('We couldn\'t authorize your payment. Please try again.', self::FILENAME);
+            $this->redirectWithNotifications($this->context->link->getModuleLink(
+                $this->module->name,
+                ControllerName::FAIL_IFRAME,
+                [
+                    'cartId' => $cartId,
+                    'secureKey' => $secureKey,
+                    'orderId' => $orderId,
+                    \Invertus\SaferPay\Config\SaferPayConfig::IS_BUSINESS_LICENCE => true,
```

Comment:
why other redirect to fail_iframe doesn't have this business_license key?

Reply:
noticed that the fail_iframe doesnt event accept bussiness_licence param. Removed and unified the redirect.